### ISSUE TITLE
No Steam/xbox menu after latest update

### ISSUE THREAD

**Issue by jackobin** [ChimeraOS/chimeraos#812]
After the latest update, the 'Steam' menu no longer appears. Only the screen is blurry, but the menu doesn't show up. The menu is still there, as it makes a sound when navigating between options.

**Comment by yb9210** [comment]
The reason for steam-patch，and  Network manger

**Comment by honjow** [comment]
I spent an entire evening solving this problem, only to discover that the cause was seemingly unrelated NetworkManager.
`sudo rm -rf /frzr_root/etc/NetworkManager`
Simply delete etc/NetworkManager, and then restart. We can delete /frzr_root/etc/NetworkManager. But I still find it hard to understand the relationship between the two.

**Comment by honjow** [comment]
> I spent an entire evening solving this problem, only to discover that the cause was seemingly unrelated NetworkManager. `sudo rm -rf /frzr_root/etc/NetworkManager` Simply delete etc/NetworkManager, and then restart. We can delete /frzr_root/etc/NetworkManager. But I still find it hard to understand the relationship between the two.

After further testing, it was found that this solution is still not perfect, and the problem may recur.
You can try the following solution:
Turn off the WiFi and then restart.

I suspect it is still related to the Steam client.

**Comment by speakradmin** [comment]
Same here. Infuriating bug cause I had Chimera set up just right and then ended up moving to Bazzite. I'll try this though as Chimera is just much more stable than Bazzite.

**Comment by honjow** [comment]
> Same here. Infuriating bug cause I had Chimera set up just right and then ended up moving to Bazzite. I'll try this though as Chimera is just much more stable than Bazzite.
> 
> Update: neither solution worked for me but I noticed the issue right after changing the hostname. Set it back to chimeraos change and restart and voila! Problem solved. Truely a strange bug.
> 
> Interestingly, on my original install I had changed the hostname months ago when first installing but the bug occured only after the applying 3.5 update. Hope that helps.

After your reminder, it seems that I have discovered that if there is a `-` in the hostname, this problem will consistently occur. I really can't understand it more and more.

**Comment by jackobin** [comment]
> Same here. Infuriating bug cause I had Chimera set up just right and then ended up moving to Bazzite. I'll try this though as Chimera is just much more stable than Bazzite.
> 
> Update: neither solution worked for me but I noticed the issue right after changing the hostname. Set it back to chimeraos change and restart and voila! Problem solved. Truely a strange bug.
> 
> Interestingly, on my original install I had changed the hostname months ago when first installing but the bug occured only after the applying 3.5 update. Hope that helps.

-Thanks. The wifi thing worked for me on the 3rd attempt

**Comment by jackobin** [comment]
> Same here. Infuriating bug cause I had Chimera set up just right and then ended up moving to Bazzite. I'll try this though as Chimera is just much more stable than Bazzite.
> 
> Update: neither solution worked for me but I noticed the issue right after changing the hostname. Set it back to chimeraos change and restart and voila! Problem solved. Truely a strange bug.
> 
> Interestingly, on my original install I had changed the hostname months ago when first installing but the bug occured only after the applying 3.5 update. Hope that helps.

-Thanks. The wifi thing worked for me on the 3rd attempt

edit: the bug came back after 2 reboots

**Comment by speakradmin** [comment]
@honjow maybe you want to update the title and bug description so it's clearer to the dev. Perhaps they could find a permanent fix

**Comment by speakradmin** [comment]
It did for me. I have not tested the `-` yet I just left it as chimeraos

**Comment by Daxiongmao87** [comment]
I'm also experiencing this issue on various software:

1. GPD Win Max 2020 (intel 1037G intel Iris graphics)
2. Custom PC, RX 6500 with i3-10100F

None of their hostnames have any dashes, however my wifi SSID does.  Is the SSID what everyone is referring to?

**Comment by jackobin** [comment]
> It did for me. I have not tested the `-` yet I just left it as chimeraos

changing host name back to CHIMERAOS the bug did not return.

Thank you all.

**Comment by Gaarindor** [comment]
Had the same issue. 
Appeared immediately after a reboot when I had my device renamed with a - in it.
I removed the - in the name and rebooted, issue went away. Currently not using chimeraos as the name.

**Comment by hperrin** [comment]
Yep, changing hostname to "chimeraos" worked. Why the f does that work?

**Comment by GRTitan** [comment]
Hello.

Created this account just to say thank you.

Just installed ChimeraOS, changed the hostname, it had a - in the name and boom, no menu.

Removed the - but kept the same hostname (not ChimeraOS) and it worked.

The issue is indeed adding a - to the hostname.

AMD 5600X+AMD 7900XTX.

**Comment by Adravil** [comment]
> Same here. Infuriating bug cause I had Chimera set up just right and then ended up moving to Bazzite. I'll try this though as Chimera is just much more stable than Bazzite.
> 
> Update: neither solution worked for me but I noticed the issue right after changing the hostname. Set it back to chimeraos change and restart and voila! Problem solved. Truely a strange bug.
> 
> Interestingly, on my original install I had changed the hostname months ago when first installing but the bug occured only after the applying 3.5 update. Hope that helps.

It helped me too, what a surprising bug hahaha 😂

**Comment by epicshepich** [comment]
> Same here. Infuriating bug cause I had Chimera set up just right and then ended up moving to Bazzite. I'll try this though as Chimera is just much more stable than Bazzite.
> 
> Update: neither solution worked for me but I noticed the issue right after changing the hostname. Set it back to chimeraos change and restart and voila! Problem solved. Truely a strange bug.
> 
> Interestingly, on my original install I had changed the hostname months ago when first installing but the bug occured only after the applying 3.5 update. Hope that helps.

How do you change the hostname if you can't use the menu?
